account_manager/src/validator/recover.rs

```diff
@@ -148,7 +148,7 @@ pub fn cli_run(matches: &ArgMatches, validator_dir: PathBuf) -> Result<(), Strin
         println!(
             "{}/{}\tIndex: {}\t0x{}",
             index - first_index,
-            count - first_index,
+            count,
```

Comment:
This is option 1:
If count = 2, the UX would be:
```
0/1
1/1
```

```suggestion
            count - 1,
```

Reply:
For the devs, this is option 1: 

If `count = 2`, the UX would be:
```
0/1
1/1 
```